### ISSUE TITLE
multiple file upload

### ISSUE THREAD

**Issue by jots** [Dogfalo/materialize#1616]
I have:
```html
 <div class="file-field input-field col s12">
          <div class="btn">
            <span>File Upload</span>
            <input type="file" multiple>
          </div>
          <div class="file-path-wrapper">
            <input class="file-path" type="text">
          </div>
        </div>
      </div>
```
when I select multiple files, only the last one shows up in the input field.  How to let them all show up?

**Comment by acburst** [comment]
Merged in 0c4864572fffeb21fe9e7e8350eb10faa5e25a79

**Comment by theevil** [comment]
I can limit the size of file in MB for example limit the file to 5 mb
